bot/client.js

```diff
@@ -94,5 +94,12 @@ client.on('message', (message) => {
   }
 });
 
-// controllers.newUserController();
+// When user plays, streams, or Listens give 5 points
+// oldMember is required due to documentation
+client.on('presenceUpdate', (oldMember) => {
+  const { id } = oldMember.user;
+  const { game } = oldMember.presence;
+  !game ? getUserPointsandUpdate(id, 5) : null;
```

Comment:
Could you explain what this line is doing? I'm not exactly sure between the whole oldMember stuff and this ternary.

Reply:
Due to the documentation oldMember is required and can not be changed. What this does is it detects when the presence changes and if its anything else other than null it will give 5 points to the user whose presence has changed.